implants/lib/host_unique/Cargo.toml

```diff
@@ -9,6 +9,9 @@ edition = "2021"
 uuid = { workspace = true, features = ["v4", "fast-rng"] }
 log = { workspace = true }
 
+[target.'cfg(target_os = "windows")'.dependencies]
+winreg = "0.55"
```

Comment:
I think we import winreg at the workspace for some of the eldritch functions.

Switch to using workspace = true to use it.

Doing this keeps all the dependencies on the same version and makes it more manageable.